### ISSUE TITLE
Tutorial 16 Meta Learning(Pytorch)

### ISSUE THREAD

**Issue by rakibMahmud** [phlippe/uvadlc_notebooks#130]
In this can you please  show the training and validation accuracy graph together for prototypical network. Then for a new image in testing task, can it identify, which class this image belongs to. If it is possible, then show us please.
